### PR TITLE
CitizensPulse

### PR DESCRIPTION
**Pull request by maniankara**
A fully distributed platform for state/city councils to know, share and feel the interests, pulse of citizens on their decisions. This is implemented with hyperledger fabric. Has a web, api-server and a mobile application

### CODE REVIEW

File: labs/citizens-pulse.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- [Hritik Gupta](https://github.com/hritikgupta)
+- [Anoop Vijayan Maniankara](https://github.com/maniankara)
+
+# Sponsor
+
+_yet to find one_
+
+# Pre-existing repository
+
+- [Hyperledger Citizens Pulse](https://github.com/maniankara/hyperledger-citizens-pulse) - **Private repository**
```
Comment (tkuhrt) [inline labs/citizens-pulse.md:63]: Link results in a 404.

File: labs/citizens-pulse.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Proposed Solution
+
+In a distributed platform like hyperledger, its possible to decentralize the ownership. This gets to a situation where the platform is by the people and for the people. Also, due to its immutable nature of the hyperledger, there is one source of truth existing in the system. The Endorsement policies handle the privacy part and Certificate-Authorities provide safe and secure platform. When the hyperledger components are shared among the NGOs/government/Universities/private-bodies, the ownership is distributed.
```
Comment (tkuhrt) [inline labs/citizens-pulse.md:17]: In this section, `hyperledger` should be replaced with `Hyperledger Fabric`

File: labs/citizens-pulse.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# Lab Name
+
+Citizens Pulse
+
+# Short Description
+
+A fully distributed platform for state/city councils to know, share and feel the interests, pulse of citizens on their decisions. This is implemented with hyperledger fabric.
```
Comment (tkuhrt) [inline labs/citizens-pulse.md:7]: In this section, `hyperledger fabric` should be replaced with `Hyperledger Fabric`

File: labs/citizens-pulse.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+Orderers and Peers are distributed and hosted across government, NGOs and private bodies.
+
+### Endorsement policy:
+
+Combination of Private data implicit collections/Key-level endorsement policies are implemented.
+
+### Overview
+
+![alt text](<https://raw.githubusercontent.com/hritikgupta/hritikgupta.github.io/master/img/citi-pulse%20(3).png>)
```
Comment (tkuhrt) [inline labs/citizens-pulse.md:31]: recommend moving the PNG file to `./labs/images/` and referencing it locally.

File: labs/citizens-pulse.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+## High-level workflow:
+
+1. Automatic enrolment with bank identification, some other reliable systems through CA or other means
```
Comment (tkuhrt) [inline labs/citizens-pulse.md:35]: `enrolment` should be `enrollment`

File: labs/citizens-pulse.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+## Demo
+
+## Installations & Bringing the Project Up
+
+[![asciicast](https://asciinema.org/a/nqKRe8LVBCG94SVoNy18RyiGy.svg)](https://asciinema.org/a/nqKRe8LVBCG94SVoNy18RyiGy)
+
+## Web User Interface
+
+![alt text](https://raw.githubusercontent.com/hritikgupta/hritikgupta.github.io/master/img/web_out.gif)
```
Comment (tkuhrt) [inline labs/citizens-pulse.md:46]: recommend moving the GIF file to `./labs/images/` and referencing it locally.

File: labs/citizens-pulse.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+## Mobile Interface
+
+<img src="https://github.com/hritikgupta/hritikgupta.github.io/blob/master/img/Screenshot_20200909-110446.png?raw=true" width="250"/> <img src="https://github.com/hritikgupta/hritikgupta.github.io/blob/master/img/Screenshot_20200909-110839.png?raw=true" width="250"/> <img src="https://github.com/hritikgupta/hritikgupta.github.io/blob/master/img/Screenshot_20200909-110844.png?raw=true" width="250"/>
```
Comment (tkuhrt) [inline labs/citizens-pulse.md:50]: recommend moving the PNG files to `./labs/images/` and referencing them locally.

File: labs/citizens-pulse.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+# Initial Committers
+
+- [Hritik Gupta](https://github.com/hritikgupta)
+- [Anoop Vijayan Maniankara](https://github.com/maniankara)
+
+# Sponsor
+
+_yet to find one_
```
Comment (tkuhrt) [inline labs/citizens-pulse.md:59]: @hyperledger-labs/lab-stewards : This is a lab that is being created from the Hyperledger mentorship program. Should we allow the mentors to be sponsors?

**Review by tkuhrt — CHANGES_REQUESTED** [review]

File: labs/citizens-pulse.md (hunk 1 above)
Comment (maniankara) [reply labs/citizens-pulse.md:63]: @tkuhrt yes, the plan is to move the repository to labs if approved. About DCO signoff, as of today, all the commits in the respository are compliant. Do you see any challenges from that approach?

Comment (tkuhrt) [reply labs/citizens-pulse.md:63]: No problems. I always like to check to make sure that repositories have the correct DCO sign offs early to ensure there are no surprises.

File: labs/citizens-pulse.md (hunk 8 above)
Comment (vipinsun) [reply labs/citizens-pulse.md:59]: I will be the sponsor. Again- we appealed to the TSC to expand the pool of sponsors.

**Review by tkuhrt — APPROVED** [review]
LGTM

**Review by vipinsun — APPROVED** [review]

**Pull request merged**